### ISSUE TITLE
serviceName in cli to avoid hickups

### ISSUE THREAD

**Issue by tvld** [feathersjs/feathers#2951]
I was breaking my head over some strange errors, 
`` "params": { "additionalProperty": "email"  }, "message": "must NOT have additional properties"}``

I was tarting to blame ... until, on Discord @marshallswain gave me the tip, in my "defense", just 5 minutes after I found it myself ;))):  I was applying hooks from one service to another service... oops.

The suggestion is now to introduce in the cli a constant ``servicePath``, one place change if you are cut and pasting the code to create a new service manually:

```ts
const servicePath = 'users' // the suggestion

export const user = async (app: Application) => {
  app.use(servicePath, new UserService(getOptions(app)), {
    methods: ['find', 'get', 'create', 'update', 'patch', 'remove'],
    events: []
  })

  app.service(servicePath).hooks(hooks)

```
The disadvantage is that it is slightly less nice to the eye... but well...just an idea ;)

**Comment by daffl** [comment]
This makes sense. I also ran into a similar problem the other day. Might defer this to the next version after the release though since any change in the generated code also requires updating the getting started guide, the CLI guide and the feathers-chat example repository.

**Comment by daffl** [comment]
Turns out this needed to be addressed in #3008 anyway since renaming a service or updating the methods between client and server was pretty cumbersome.
